Tools/rocprof-systems/.gitignore

```
@@ -0,0 +1,5 @@
+rocprof-systems_matmul
+rocprof-systems_matmul.inst
+rocprof-systems_matmul.papi.inst
+rocprof-systems_matmul_userapi
+rocprof-systems_matmul_userapi.inst
```

```suggestion
rocprof-systems_matmul_userapi.inst

```
Ending file with a new line.